integration/docker/entrypoint.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [[ $# -ne 1 ]]; then
+  echo 'expected one argument: "master", "worker", or "proxy"'
+  exit 1
+fi
+
+service=$1
+
+home=/opt/$(ls /opt | grep alluxio)
+cd ${home}
+
+for keyvaluepair in $(env | grep "ALLUXIO_"); do
```

Comment:
There will be a conflict with actual environment variables that are prefixed with ALLUXIO_*, like ALLUXIO_HOSTNAME, ALLUXIO_CLASSPATH, ALLUXIO_RAM_FOLDER, etc: http://www.alluxio.org/docs/master/en/Configuration-Settings.html#environment-variables

This should check for a different prefix?